vector/src/main/java/im/vector/services/EventStreamServiceX.kt

```diff
@@ -200,7 +208,12 @@ class EventStreamServiceX : VectorService() {
 
         when (action) {
             ACTION_START,
-            ACTION_GO_TO_FOREGROUND ->
+            ACTION_GO_TO_FOREGROUND -> {
+
+                //We are back in foreground, we can sync
+                mSession?.syncDelay = 0
+                mSession?.syncTimeout = 30000
```

Comment:
Where does this value `30000` come from?

Reply:
recommended value for riot